### ISSUE TITLE
Bulk actions

### ISSUE THREAD

**Issue by jlixfeld** [respawner/peering-manager#59]
### Issue type
[x] Feature request 
[ ] Bug report      
[ ] Documentation   

### Environment
* Python version:   3.5.3
* Peering Manager version:   master

### Description

In the below example, it would be very helpful to see the status of each of these sessions (Established w/refix received or Idle/Active with reason (ie: down due to max-prefix violation (#53 ))) in this view, but also be able to apply bulk actions, like Disable selected sessions and apply a bulk comment to all sessions.

For instance, many of these sessions below are Idle because the far side hasn't set them up yet, but to prevent alarms, I'd like to select the ones that are idle, bulk disable them and add a bulk comment to each of them to say "Nov 7th - imported, waiting for far side to configure them").  Currently, I need to click Edit on each one, check to see if it's not Established, if not, disable the session, add a commend and then click Update

<img width="1399" alt="screen shot 2018-11-07 at 7 22 42 am" src="https://user-images.githubusercontent.com/17339017/48131540-6c810c80-e25e-11e8-85ef-d9b4b5f79439.png">

**Comment by respawner** [comment]
We'll present a bulk edit view based on what the user has selected. Making the code as generic as possible would allow this view to be used with several models.

**Comment by jlixfeld** [comment]
This is a great commit!  Thank you!!

Could it be extended to the global view as well?  IE:  In the image attached above, bulk edit for a network that is present across multiple IXs.

One thing I just noticed too is cases where the number of sessions spans multiple pages.  It might be helpful to have all sessions appear one page to ensure they are all visible for selection.

**Comment by respawner** [comment]
Yeah the feature will be extended to other views as well in the coming days. Expect bulk edit on a bunch of field for AS, routers, IX, etc…

Your second question will be addressed in #66.
